src/server/scripts/Northrend/IcecrownCitadel/boss_professor_putricide.cpp

```diff
@@ -1008,9 +1008,11 @@ class spell_putricide_unstable_experiment : public SpellScriptLoader
                     break;
                 }
 
-            if (Aura* aura = target->GetAura(uint32(GetSpellInfo()->Effects[stage].CalcValue())))
-                if (aura->GetOwner() == target) // avoid assert(false) at any cost
-                    aura->UpdateOwner(5000, target); // update whole aura so previous periodic ticks before refreshed by new one
+            if (target != nullptr) {
```

Comment:
no curly/php braces please

Reply:
you mean that I should make them like this
```C++
if ()
{

}
```

or should I remove them totally?

Reply:
yes, that's how brackets are set everywhere else

Reply:
I remember that it is present also in the code guide lines but I totally forgot to respect that because I was just editing the code where there were no braces